detector/wordpress.go

```diff
@@ -197,17 +215,25 @@ func extractToVulnInfos(pkgName string, cves []WpCveInfo) (vinfos []models.VulnI
 			})
 		}
 
+		v3score := 0.0
+		if vulnerability.Cvss.Score != "" {
+			v3score, _ = strconv.ParseFloat(vulnerability.Cvss.Score, 64)
+		}
+
 		for _, cveID := range cveIDs {
 			vinfos = append(vinfos, models.VulnInfo{
 				CveID: cveID,
 				CveContents: models.NewCveContents(
 					models.CveContent{
-						Type:         models.WpScan,
-						CveID:        cveID,
-						Title:        vulnerability.Title,
-						References:   refs,
-						Published:    vulnerability.CreatedAt,
-						LastModified: vulnerability.UpdatedAt,
+						Type:          models.WpScan,
+						CveID:         cveID,
+						Title:         vulnerability.Title,
```

Comment:
The information in the Description may also be useful.
```suggestion
						Title:         vulnerability.Title,
						Summary:       vulnerability.Description,
```

Reply:
Thank you.
The description is a field exclusive to Enterprise, so I have made corrections including a null check.

```go
// WpCveInfo is for wpscan json
type WpCveInfo struct {
        ....
	Description   *string    `json:"description"` // Enterprise only
        ....
}
```